dds_web/api/user.py

```diff
@@ -89,12 +91,33 @@ def invite_user(args):
             raise ddserr.InviteError(message=valerr.messages)
 
         # Create URL safe token for invitation link
-        s = itsdangerous.URLSafeTimedSerializer(flask.current_app.config["SECRET_KEY"])
-        token = s.dumps(new_invite.email, salt="email-confirm")
+        TKEK = "bogus"
+        # TODO change to real TKEK.
+
+        sensitive_content = json.dumps({"invited_email": new_invite.email, "TKEK": TKEK})
```

Comment:
this just slipped in. let's remove it (unused)

Reply:
Very observant of you! Fixed!